.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 dist
+.idea
```

Comment:
@vacekj can you remove this .idea rule and put it in your local global gitignore please?

Reply:
sure, done.